pages/linux/timeout.md

```diff
@@ -0,0 +1,11 @@
+# timeout
+
+> Run a command with a time limit.
+
+- Run sleep command (for 10 seconds) but kill it after 3 seconds:
+
+`timeout 3s sleep 10`
+
+- Run sleep command (for 10 seconds) but kill it when interruptted or after 5 seconds:
```

Comment:
Typo in interruptted.

Also a question here - a running process should automatically quit when the SIGINT signal is sent. What is the need to specify it here with timeout ?

Reply:
The timeout limit is required in case no signal is sent and some process could run indefinitely.

Reply:
My point was that a process will automatically quit when a SIGINT is received. So, is there a benefit of adding this example in the timeout page ?